### PR TITLE
Added Icons for Meta's Threads.net Platform

### PR DESCRIPTION
**Pull request by AayushGithub**
Threads is an American social media and social networking service owned and operated by Meta Platforms. It operates in a similar manner to other microblogging platforms such as Twitter. Users have the ability to post and share text, images, and videos, as well as interact with other users' posts through replies, reposts, and likes. Threads is closely integrated with Instagram, requiring users to have an Instagram account and use the same handle. The application is available on iOS and Android devices with full functionality, while the web version offers limited functionality.

### CODE REVIEW

**Comment by AayushGithub** [comment]
I have no idea why its failing

**Comment by amio** [comment]
Thanks @AayushGithub 

It failed as Node.js 12 is deprecated on Vercel. Just updated the configuration, it's fine now.

File: icons/threads-color.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...ffset="1" stop-color="#feda77"/></linearGradient></defs><path d="M119.16 82.5c-.76-.37-1.54-.72-2.33-1.06-1.37-25.32-15.21-39.81-38.43-40h-.32c-13.89 0-25.45 5.93-32.56 16.72L58.3 67c5.31-8.06 13.65-9.78 19.79-9.78h.21c7.65.05 13.42 2.27 17.15 6.61 2.72 3.15 4.54 7.52 5.44 13a97.8 97.8 0 0 0-22-1.06C56.85 77 42.66 89.92 43.61 107.81a27.6 27.6 0 0 0 12.73 22A39.4 39.4 0 0 0 80 135.75c11.55-.64 20.61-5 26.93-13.1 4.8-6.12 7.83-14.05 9.17-24a28.29 28.29 0 0 1 11.84 13c3.83 8.93 4.06 23.61-7.92 35.58-10.49 10.48-23.11 15-42.17 15.15-21.15-.15-37.14-6.93-47.54-20.15-9.7-12.47-14.73-30.35-14.91-53.23.18-22.88 5.21-40.76 15-53.14 10.4-13.22 26.39-20 47.54-20.15 21.3.15 37.57 7 48.36 20.25 5.3 6.51 9.29 14.7 11.92 24.25l15-4a81.07 81.07 0 0 0-15-30.29C124.26 8.91 104 .18 77.94 0h-.1c-26 .18-46.05 8.94-59.49 26C6.38 41.24.21 62.41 0 88.94v.12c.21 26.53 6.38 47.7 18.35 62.91 13.44 17.09 33.46 25.85 59.49 26h.1c23.14-.16 39.45-6.22 52.89-19.64 17.58-17.57 17-39.58 11.25-53.1-4.15-9.66-12.08-17.53-22.92-22.73m-39.95 37.56c-9.68.55-19.73-3.8-20.23-13.1-.37-6.9 4.91-14.6 20.83-15.52 1.82-.1 3.61-.15 5.36-.15a76 76 0 0 1 16.11 1.63c-1.84 22.91-12.59 26.62-22.07 27.14" style="fill:url(#a)"/></svg>
```
Comment (amio) [inline icons/threads-color.svg:1]: we don't need color version

File: icons/threads-bw.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...29 2.932 3.405 4.893 8.111 5.864 14.05-7.314-1.243-15.224-1.626-23.68-1.14-23.82 1.371-39.134 15.264-38.105 34.568.522 9.792 5.4 18.216 13.735 23.719 7.047 4.652 16.124 6.927 25.557 6.412 12.458-.683 22.231-5.436 29.049-14.127 5.178-6.6 8.453-15.153 9.899-25.93 5.937 3.583 10.337 8.298 12.767 13.966 4.132 9.635 4.373 25.468-8.546 38.376-11.319 11.308-24.925 16.2-45.488 16.351-22.809-.169-40.06-7.484-51.275-21.742C35.236 139.966 29.808 120.682 29.605 96c.203-24.682 5.63-43.966 16.133-57.317C56.954 24.425 74.204 17.11 97.013 16.94c22.975.17 40.526 7.52 52.171 21.847 5.71 7.026 10.015 15.86 12.853 26.162l16.147-4.308c-3.44-12.68-8.853-23.606-16.219-32.668C147.036 9.607 125.202.195 97.07 0h-.113C68.882.194 47.292 9.642 32.788 28.08 19.882 44.485 13.224 67.315 13.001 95.932L13 96v.067c.224 28.617 6.882 51.447 19.788 67.854C47.292 182.358 68.882 191.806 96.957 192h.113c24.96-.173 42.554-6.708 57.048-21.189 18.963-18.945 18.392-42.692 12.142-57.27-4.484-10.454-13.033-18.945-24.723-24.553ZM98.44 129.507c-10.44.588-21.286-4.098-21.82-14.135-.397-7.442 5.296-15.746 22.461-16.735 1.966-.114 3.895-.169 5.79-.169 6.235 0 12.068.606 17.371 1.765-1.978 24.702-13.58 28.713-23.802 29.274Z" /></svg>
```
Comment (amio) [inline icons/threads-bw.svg:1]: the color is not quite right
image

**Comment by AayushGithub** [comment]
@amio Thanks for looking into this. I have updated it to reflect only 1 white svg

**Comment by AayushGithub** [comment]
@amio ^

**Comment by AayushGithub** [comment]
@amio checking in again

**Comment by amio** [comment]
Sorry for missing the updates. That's perfect, thanks @AayushGithub !

**Pull request merged**